test/BotwinModuleTests.cs

```diff
@@ -174,5 +174,28 @@ public BotwinModuleTests()
             var body = await response.Content.ReadAsStringAsync();
             Assert.True(body.Contains("After"));
         }
+        
+        [Fact]
+        public async Task Should_return_GET_requests_with_parsed_querystring()
+        {
+            const int idToTest = 69; 
+            var response = await this.httpClient.GetAsync($"/querystring?id={idToTest}");
+
+            var body = await response.Content.ReadAsStringAsync();
+
+            Assert.Equal(200, (int)response.StatusCode);
+            Assert.True(body.Contains($"Managed to parse an int {idToTest}"));
+        }
+        
+        [Fact]
+        public async Task Should_return_GET_requests_with_multiple_parsed_querystring()
```

Comment:
Worth a test if you do `?id=1,2,3`?

Reply:
Not tested, but I imagine it would fail on the parsing, because the query collection would return that as one value.

Reply:
not sure it would because query[id] returns ienumerable<string> so i'd expect that with 1,2,3 in it

Reply:
That returns one value. I don't think we should go down the route of splitting that manually either

Reply:
Thinking about it, another overload to handle that would be quite nice.